perfschema/init.go

```diff
@@ -0,0 +1,1411 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package perfschema
+
+import (
+	"sync/atomic"
+
+	"github.com/juju/errors"
+	"github.com/pingcap/tidb/column"
+	"github.com/pingcap/tidb/field"
+	"github.com/pingcap/tidb/model"
+	"github.com/pingcap/tidb/mysql"
+	"github.com/pingcap/tidb/util/charset"
+	"github.com/pingcap/tidb/util/codec"
+	"github.com/pingcap/tidb/util/types"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/storage"
+)
+
+// CREATE TABLE if not exists performance_schema.setup_actors (
+// 		HOST			CHAR(60) NOT NULL  DEFAULT '%',
+// 		USER			CHAR(32) NOT NULL  DEFAULT '%',
+// 		ROLE			CHAR(16) NOT NULL  DEFAULT '%',
+// 		ENABLED			ENUM('YES','NO') NOT NULL  DEFAULT 'YES',
+// 		HISTORY			ENUM('YES','NO') NOT NULL  DEFAULT 'YES');
+func (ps *perfSchema) buildSetupActorsModel() {
+	tbName := TableSetupActors
```

Comment:
How about extract to a function?
Seems u can use table, cols as variables, then the code will be shorter and clearer.